ui/packages/catalog-ui-search/src/main/webapp/react-component/button/split-button/index.tsx

```diff
@@ -31,55 +31,40 @@ type State = {
 const Root = styled.div`
   position: relative;
   overflow: none;
-  padding: 0;
   margin: 0;
-  display: flex;
+  white-space: nowrap;
+  background-color: ${props => props.theme.primaryColor};
 `
 const Menu = styled.div`
   position: absolute;
   top: ${props => props.theme.minimumButtonSize};
-  padding: 0;
+  padding-top: ${props => props.theme.minimumSpacing};
+  padding-bottom: ${props => props.theme.minimumSpacing};
```

Comment:
✏️ Could also right these two lines as: 
```suggestion
  padding: ${props => props.theme.minimumSpacing} 0 ${props => props.theme.minimumSpacing};
```

Reply:
IMO the explicit attributes are more readable if only two sides are being set